### PR TITLE
Make title screen

### PR DESCRIPTION
**Pull request by buzzingnat**

### CODE REVIEW

**Comment by buzzingnat** [comment]
Ready to be merged in.

File: app/scenes/fileSelect/renderFileSelect.ts
Comment (buzzingnat) [inline app/scenes/fileSelect/renderFileSelect.ts:1]: Formerly "renderTitle"

File: app/scenes/fileSelect/updateFileSelect.ts
Comment (buzzingnat) [inline app/scenes/fileSelect/updateFileSelect.ts:1]: formerly "updateTitle"

**Review by lancehalberd — APPROVED** [review]
Thanks! I'll review and fix any remaining issues after merging.

**Pull request merged**